README.md

```diff
@@ -30,3 +30,50 @@ $ gulp
 Then point your browser to [http://localhost:8000](http://localhost:8000/).
 
 Note: The site is built in the `dist/app` folder.
+
+## Building for Cordova
+
+Run `npm install -g cordova` to install Cordova, and make sure you have the Android
+emulator installed. Then, to build this app as a Cordova app, run:
+
+```bash
+gulp build
+cd dist
+cordova create cordova
+rm -r cordova/www/*
+cp -r app/* cordova/www/
+rm cordova/www/cordova.js
+cd cordova
+cordova plugin add https://github.com/michielbdejong/SecureHTTP.git
+cordova platform add android
+cordova build android
+cordova emulate android
+```
```

Comment:
Do you think you can make this a gulp task in the `gulpfile.js` instead?
The command to build would just have to be `gulp cordova`.